packages/entity-server/src/app/createApp.ts

```diff
@@ -38,8 +38,11 @@ export function createApp(models: EntityServerModelRegistry) {
   app.use((req: Request, res: Response, next: NextFunction) => {
     req.models = models;
     req.authenticator = authenticator;
-    req.context = {};
-    res.context = {};
+
+    const context = {}; // context is shared between request and response
+    req.context = context;
+    res.context = context;
```

Comment:
Decided to go with one shared context between request and response. My feeling was that the distinction between what belonged where was rather arbitrary... I like this way but can understand if others find it a little unexpected